Dockerfile.rhel7

```diff
@@ -2,13 +2,12 @@ FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.21 AS
 WORKDIR /go/src/github.com/openshift/service-ca-operator
 COPY . .
 ENV GO_PACKAGE github.com/openshift/service-ca-operator
-RUN make build \
-    && make tests-ext-build \
-    && gzip test/tests-extension/bin/service-ca-operator-tests-ext
+RUN make build --warn-undefined-variables \
+    && gzip service-ca-operator-tests
```

Comment:
this is wrong folder should be service-ca-operator-tests-ext and then you compile with make build then you should have service-ca-operator-tests-ext binary

Reply:
Updated.